src/h3/mod.rs

```diff
@@ -582,6 +589,43 @@ impl Connection {
         Ok(())
     }
 
+    fn send_grease_frames(
+        &mut self, conn: &mut super::Connection, stream_id: u64,
+    ) -> Result<()> {
+        let mut d = [42; 128];
+
+        let mut b = octets::Octets::with_slice(&mut d);
+        b.put_varint(0)?;
+        b.put_u8(0xb)?;
```

Comment:
It's not really greasing if you hardcode the values...

Reply:
Yup, I have protyped a better version in the draft 19 branch because the grease values are different and common across all greasing.

Reply:
So I propose landing this change, or waiting to incorporate this change into the draft 19 /20 update.